src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -2018,12 +2018,14 @@ ImmutableArray<IInterpolatedStringContentOperation> createNonHandlerInterpolated
                 return builder.ToImmutableAndFree();
             }
 
-            ImmutableArray<IInterpolatedStringContentOperation> createHandlerInterpolatedStringContent(ImmutableArray<(bool IsLiteral, bool HasAlignment, bool HasFormat)> positionInfo)
```

Comment:
Nit: not necesarily as part of this PR, but consider creating a struct for this tuple for ease of reading/understanding, especially when its somewhat divorced from the original context.

Reply:
Sure. IOperation changes will come after this, so I can do this in the next PR